packages/react-ui-components/src/components/SchemaForm/Widgets/CheckboxesWidget/CheckboxesWidget.tsx

```diff
@@ -43,7 +44,7 @@ const CheckboxesWidget = ({ schema, label, id, disabled, options, value, require
 			{(get(uiSchema, ['ui:title']) || schema.title || label) && (
 				<KvFormLabel id={`${id}-title`} label={get(uiSchema, ['ui:title']) || schema.title || label} required={required} />
 			)}
-			<KvRadioButtonGroup buttons={buttons} selectedButtons={selectedButtons} onCheckedChange={onCheckedChange} disabled={disabled} />
```

Comment:
You don't need the disabledButtons here anymore.

Reply:
The `disabledButtons` give the ability to have one of the buttons disable and rest not. Even though it is not a foreseeable use case on the short term, it makes the overall component more flexible.

What are your thoughts on this?

Reply:
Oh sorry, I thought this was on a storybook story. It makes total sense 😄